test/functional/api/vim_spec.lua

```diff
@@ -1036,5 +1036,33 @@ describe('api', function()
       eq(expected, nvim("list_uis"))
     end)
   end)
-
+  describe('nvim_get_keyboard_mods_state', function()
+    it('on linux either xlib11 is available or not', function()
+      if os_name() == 'unix' then
+        local status, rv = pcall(nvim, "get_keyboard_mods_state")
+        if status == true then
+          eq(rv, {CapsLock=false, NumLock=false, ScrollLock=false})
+        else
+          eq("Unable to load ", string.match(rv, "Unable to load *"))
+        end
+      end
+    end)
+    it('on windows there is always an acceptance', function()
+      if os_name() == 'windows' then
```

Comment:
And I would really suggest to have *one* test as they are mutually exclusive and are supposed to share a code for checking return value.